README.md

```diff
@@ -284,22 +266,6 @@ Use this command to print the version of elastic-package that you have installed
 
 
 
-### GitHub authorization
-
-The `promote` and `publish` commands require access to the GitHub API to open pull requests or check authorized account data.
-The tool uses the GitHub token to authorize user's call to API. The token can be stored in the `~/.elastic/github.token`
-file or passed via the `GITHUB_TOKEN` environment variable.
-
-Here are the instructions on how to create your own personal access token (PAT):
-https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token
-
-Make sure you have enabled the following scopes:
-* `public_repo` — to open pull requests on GitHub repositories.
-* `read:user` and `user:email` — to read your user profile information from GitHub in order to populate pull requests appropriately.
-
-After creating or modifying your personal access token, authorize the token for
-use of the Elastic organization: https://docs.github.com/en/github/authenticating-to-github/authenticating-with-saml-single-sign-on/authorizing-a-personal-access-token-for-use-with-saml-single-sign-on
-
```

Comment:
As elastic-package should not create any PRs, I think this section is not needed anymore. Is there any other usage for this token ? @elastic/ecosystem